account_invoice_accrual/README.rst

```diff
@@ -0,0 +1,31 @@
+Account Invoice Accrual
```

Comment:
Can you add the license badge in README ?

see https://github.com/OCA/maintainer-tools/tree/master/template/module